pkg/docker/cockpit-atomic-storage

```diff
@@ -195,12 +180,17 @@ def get_driver_info():
         conf = open(ds_conf_file, "r").read()
     else:
         conf = ""
-    m = re.search("dm.thinpooldev=([^ ]*)", conf)
+    m = re.search("--storage-driver ([^ ]*)", conf)
     if m:
-        driver = "thinpool"
-        (lvol, vgroup) = get_dev_lvol_and_vgroup(m.group(1))
+        driver = m.group(1)
+    else:
+        driver = "overlay2"
```

Comment:
I wonder if this should check `grep -q overlay /proc/filesystems` to ensure the running kernel/distro actually has overlayfs support?

Reply:
Hmm. It should figure out the real default of Docker, without having to start docker.  But I think this isn't relevant right now.  In a way, this just means "not-devicemapper"...